providers/aws/services/iam/iam_support_role_created/iam_support_role_created_test.py

```diff
@@ -0,0 +1,44 @@
+from json import dumps
+from unittest import mock
+
+from boto3 import client
+from moto import mock_iam
+
+
+class Test_iam_support_role_created:
```

Comment:
Include tests for the other use cases like no roles with the support role attached.

Reply:
Done